content/blog/automated-releases-part-two.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Full automation of release with GitHub Actions and Conventional Commits for non-JS projects - part 2"
+date: 2020-03-24T06:00:00+01:00
+type: blog
+featured: true
+tags:
+  - GitHubActions
+  - Release
+cover: /images/posts/robot2.jpg
+weight: 100
+authors:
+  - name: Lukasz Gornicki
+    photo: /images/avatars/lpgornicki.png
+    link: https://twitter.com/derberq
+    byline: AsyncAPI Maintainer and Dev Comm Keeper
+---
+
+> tl;dr
+[Here](/blog/automated-releases/) you can find the first blog post about automated releasing. The purpose of this blog post is to show how you can do the same automation in non-JavaScript projects. Even if JavaScript community created tooling, you can still use it in other projects and don't freak out.
+
+This post and the [previous one](/blog/automated-releases/) come from our experience we gained when working on full automation for all tools maintained by [AsyncaPI Initiative](https://github.com/asyncapi/asyncapi/).
+
+> [AsyncAPI](https://github.com/asyncapi/asyncapi/) is a specification that you use to create machine-readable definitions of your event-driven APIs. 
+
+The previous post focused on JavaScript as the first library that we automated was our [generator](https://github.com/asyncapi/generator/). It covered publishing to NPM and usage of the JavaScript community ecosystem. Now we have automation rolled out to all our libraries, Go-written too.
+
+## What I need to automate release?
```

Comment:
```suggestion
## What do I need to automate on releases?
```

Reply:
`on releases` sounds strange to me

`What do I need to automate releases?` is better

Reply:
Sounds good too 👍

Reply:
What's needed to automate a release?

Reply:
I actually did `What do I need to automate releases?` to write directly to the reader

Reply:
Sounds good then 👍